pkg/operator/client/types/types.go

```diff
@@ -0,0 +1,48 @@
+package types
+
+// TODO: can this type be removed? all fields exist in HelmChart
```

Comment:
Left this note here since since the `OrderedDir` type mimics the `v1beta1.HelmChart` type, but one is used to install and the other is used to uninstall.

Reply:
in the most recent changes introduced in 1.98.2, OrderedDir is also used for uninstall. so this can probably be removed

Reply:
refactored getSortedCharts to handle v1beta1 and v1beta2 charts